plugins/lookup/onepassword_doc.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: onepassword_doc
+    author:
+      - Sam Doran (@samdoran)
+    requirements:
+      - C(op) 1Password command line utility. See U(https://support.1password.com/command-line/)
+    short_description: Fetch one or more documents stored in a 1Password vault
+    version_added: "9.0.0"
+    description:
+      - P(community.general.onepassword_raw#lookup) wraps C(op) command line utility to fetch document(s) from 1Password.
+    options:
+      _terms:
+        description: identifier(s) (UUID or name, case-insensitive) of documents(s) to retrieve.
+        required: true
+      master_password:
+        description: The password used to unlock the specified vault.
+        aliases: ['vault_password']
+      section:
+        description: Item section containing the field to retrieve (case-insensitive). If absent will return first match from any section.
+      subdomain:
+        description: The 1Password subdomain to authenticate against.
+      domain:
+        description: Domain of 1Password.
+        default: '1password.com'
+        type: str
+      account_id:
+        description: The account ID to target.
+        type: str
+      username:
+        description: The username used to sign in.
+      secret_key:
+        description: The secret key used when performing an initial sign in.
+      service_account_token:
+        description:
+          - The access key for a service account.
+          - Only works with 1Password CLI version 2 or later.
+        type: string
```

Comment:
Some of the above parameters sound like they probably also are present for the other `onepassword*` lookups. Maybe it's worth moving them to a docs fragment?

Reply:
I recall trying to do that and didn't think docs fragments worked for lookup plugins, only modules. It would absolutely be good to have a docs fragment for these plugins since they do share a common set of parameters.

Reply:
I'm not sure whether they "always" worked for plugins (or for how long "always" is), but since Ansible 2.9 they definitely also worked for plugins :)

Reply:
Awesome. I'll add a docs fragment then.